community/modules/scripts/htcondor-install/files/autoscaler.py

```diff
@@ -253,17 +253,14 @@ def scale(self):
         current_target = responseGroupInfo["targetSize"]
         print(f"Current MIG target size: {current_target}")
 
-        being_born_states = ["CREATING", "RECREATING", "VERIFYING"]
-        being_born_filters = [ f"currentAction = \"{state}\"" for state in being_born_states ]
-        being_born_combined_filter = ' OR '.join(being_born_filters)
-        reqCreatingInstances = self.instanceGroupManagers.listManagedInstances(
+        reqModifyingInstances = self.instanceGroupManagers.listManagedInstances(
             project=self.project,
             **self.zoneargs,
             instanceGroupManager=self.instance_group_manager,
-            filter=being_born_combined_filter,
+            filter="currentAction != \"NONE\"",
```

Comment:
NONE is healthy or unhealthy? Comment in description could be interpreted either way. 

From context I believe NONE is healthy.

I suggest adding comment here.